packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -462,6 +499,14 @@ export default {
 .c-megaModal-title {
     margin: 0 f.spacing(f) 0 0;
 
+    &.c-megaModal-hasBackBtn {
+        margin: 0 0 0 f.spacing(f);
+    }
+
+    &.c-megaModal-modeRTL {
+        margin: 0 f.spacing(f) 0 0;
+    }
+
```

Comment:
Same here – will become:

```suggestion
    .c-megaModal--hasBackBtn & {
        margin: 0 0 0 f.spacing(f);
    }

    .c-megaModal--modeRTL & {
        margin: 0 f.spacing(f) 0 0;
    }

```

Reply:
Just a note – you'll likely need to remove the outer `.c-megaModal` in the sass here (it shouldn't be needed anyway, it's just making the styles more specific for no reason by the looks of it)

Reply:
Did try this and it seems to break some of the colour styles.